.github/ignore-words

```diff
@@ -0,0 +1,4 @@
+brin
```

Comment:
It's hard to understand from the name what this file does. Can it be renamed into something like `codespell-ignore`?

Reply:
Renamed to `codespell-ignore-words`.